build.assets/Dockerfile

```diff
@@ -51,6 +52,11 @@ RUN (groupadd ci --gid=$GID -o && useradd ci --uid=$UID --gid=$GID --create-home
 RUN (curl -L https://github.com/coreos/etcd/releases/download/v3.3.9/etcd-v3.3.9-linux-amd64.tar.gz | tar -xz && \
      cp etcd-v3.3.9-linux-amd64/etcd* /bin/)
 
+# Install Google Cloud SDK.
+RUN (echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && \
+     curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add - && \
+     apt-get update -y && apt-get install google-cloud-sdk google-cloud-sdk-firestore-emulator -y)
+
```

Comment:
Any reason you're adding this to the beginning of the Dockerfile and not appending to the end? Logically we should add it to the end to avoid rebuilding as much of the image as possible.

Reply:
I wanted to keep it close to the etcd installation step. I can move it to the end if that should make things more efficient.